src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -3648,8 +3667,8 @@ private unsafe void ParseRemaining()
                 }
             }
 
-            //NB: A string must have at least 3 characters and at least 1 before ':'
-            if (idx + 2 >= length || end == idx)
+            //NB: A string must have at least 3(Windows)/1(Unix) characters and at least 1 before ':'
```

Comment:
Nit: I assume the 3/1 chars being referred to here are:
- On Unix the opening slash
- On Windows the opening drive letter, colon, and slash?
If so, might be worth clarifying that further in the comment. If not, I'm obviously not understanding, so it'd be worth spelling it out :)